tetrapod_backend/db/connection.py

```diff
@@ -8,20 +8,20 @@
 
 def connect(config, timeout=3000):
     global MONGO
-    host = config.get("host")
-    port = config.get("port")
-    user = config.get("user")
-    pwd = config.get("pass")
-    db_name = config.get("db")
-    conn_str = f"mongodb://{user}:{pwd}@{host}:{port}/" if(not(user == None or pwd == None) and not(user == "" or pwd == "")) else f"mongodb://{host}:{port}/"
+    # host = config.get("host")
+    # port = config.get("port")
+    # user = config.get("user")
+    # pwd = config.get("pass")
+    # db_name = config.get("db")
+    # conn_str = f"mongodb://{user}:{pwd}@{host}:{port}/" if(not(user == None or pwd == None) and not(user == "" or pwd == "")) else f"mongodb://{host}:{port}/"
     try:
-        _LOGGER.info(f"[init] Connecting to {conn_str}")
-        MONGO = MongoClient(conn_str, serverSelectionTimeoutMS=timeout)[db_name]
+        # _LOGGER.info(f"[init] Connecting to {conn_str}")
+        MONGO = MongoClient("mongodb+srv://TetrapodShop:NTOUTetrapod@cluster0.keacd.mongodb.net")['Tetrapod']
```

Comment:
DO NOT EXPOSE PASSWORD ON GITHUB
to prevent this, check you git status before commit and push, make sure you don't add tetrapod_backend/db/connection.py in your staging files